Droidcon-Boston/app/src/test/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailViewModelTest.kt

```diff
@@ -0,0 +1,168 @@
+package com.mentalmachines.droidcon_boston.views.detail
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.mentalmachines.droidcon_boston.R
+import com.mentalmachines.droidcon_boston.data.FirebaseDatabase
+import com.mentalmachines.droidcon_boston.data.Schedule
+import com.mentalmachines.droidcon_boston.data.UserAgendaRepo
+import com.mentalmachines.droidcon_boston.firebase.FirebaseHelper
+import com.mentalmachines.droidcon_boston.firebase.FirebaseHelperRobot
+import com.mentalmachines.droidcon_boston.utils.NotificationUtils
+import org.junit.Assert.*
```

Comment:
Detekt is going to flag these wildcard imports....turn them off in your Android Studio setup

Reply:
It looks like I might have branched out before the circleci stuff was merged in. That's why I didn't see this and also why there's no CI checkmark next to this PR. Should I rebase before we merge it then?